xbmc/platform/android/network/NetworkAndroid.cpp

```diff
@@ -246,6 +251,13 @@ CNetworkAndroid::~CNetworkAndroid()
     delete intf;
   for (auto intf : m_oldInterfaces)
     delete intf;
+
+  if (CJNIBase::GetSDKVersion() >= 24)
+  {
+    CJNIConnectivityManager connectivityManager(
```

Comment:
const?

Reply:
It cannot be `const` since it has member functions that are not declared as `const`